src/styles/index.scss

```diff
@@ -2,2 +1,287 @@
+// #region card mixin
+$primary-color: #00acdc;
+$secondary-color: #616070;
+$triple-color: #060b35;
+$four-color: #fff;
+$font-size-sm: 10px;
+$font-size-md: 12px;
+$font-size-lg: 16px;
+
+// #endregion
+// #region header
+:root {
+  --bg-color: #00acdc;
+  --count-size-monitor: 1;
+}
+
+* {
+  font-family: Roboto, sans-serif;
   margin: 0;
+  padding: 0;
+
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  background-color: #fff;
+  justify-content: space-between;
+  height: 60px;
+  padding-right: 50px;
+
+  &__logo {
+    height: 60px;
+  }
+
+  &__logo-moyo {
+    margin-top: 10px;
+    margin-left: 50px;
+    height: 40px;
+  }
+}
+
+.nav {
+  display: flex;
+  justify-content: end;
+  align-items: center;
+  height: 60px;
+
+  &__list {
+    display: flex;
+    list-style: none;
+    padding: 0;
+    flex-wrap: nowrap;
+  }
+
+  &__item {
+    position: relative;
+    margin-right: 20px;
+    padding: 0;
+
+    &-button {
+      display: inline-block;
+      text-align: center;
+      text-decoration: none;
+      color: #060b35;
+      height: 100%;
+      line-height: 60px;
+      position: relative;
+      cursor: pointer;
+    }
+
+    &-button:hover,
+    .is-active {
+      color: var(--bg-color);
+    }
+
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+}
+
+.is-active::after {
+  content: '';
+  position: absolute;
+  left: 0;
+  bottom: 0;
+  width: 100%;
+  height: 4px;
+  background-color: var(--bg-color);
+  border-radius: 8px;
+  opacity: 1;
+}
+
+// #endregion
+// #region main
+.main {
+  display: flex;
+  justify-content: center;
+  height: 100vh;
+
+  &__container {
+    display: grid;
+    grid-template-columns: repeat(var(--count-size-monitor), 200px);
+
+    grid-template-rows: repeat(406px);
+
+    @media (min-width: 488px) {
+      --count-size-monitor: 2;
+    }
+
+    @media (min-width: 768px) {
+      --count-size-monitor: 3;
+    }
+
+    @media (min-width: 1024px) {
+      --count-size-monitor: 4;
+    }
+
+    gap: 46px 48px;
+    margin: 50px 40px;
+
+    // margin-left: 40px;
+    // margin-right: 40px;
+
+    height: fit-content;
+    width: fit-content;
+  }
+}
+
+// #endregion
+// #region card
+.card {
+  width: 200px;
+  background-color: $four-color;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+  transition: all 0.3s;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+    background-repeat: no-repeat;
+    background-size: auto;
+  }
+
+  &__description {
+    width: 166px;
+    height: 36px;
+
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $triple-color;
+
+    &--margin--top {
+      margin-top: 40px;
+    }
+  }
+
+  &:hover {
+    .card__description {
+      color: #34568b;
+    }
+
+    transform: scale(120%);
```

Comment:
The 'transform: scale(120%)' property may cause layout issues due to the large scaling factor. Consider using a smaller scale value to avoid potential overflow or misalignment.